python-frontend/src/main/java/org/sonar/python/types/TypeShed.java

```diff
@@ -80,6 +85,37 @@ public void visitFunctionDef(FunctionDef functionDef) {
     return builtins;
   }
 
+  private static Set<Symbol> typingModuleSymbols() {
```

Comment:
Do you think we could have a test similar to those for `Union` & `Optional` which highlight the different behaviours for Python 3 specific symbols, Python 2 specific symbols and overlaps?

Reply:
Done in 11bf0f4bf2fec896f7538e525acd152168006b73
Note that there are no Python2 symbols for typing: they all overlap with Python3 symbols